.travis.yml

```diff
@@ -1,14 +1,33 @@
 language: go
 
+services:
+  - docker
+
 install:
   - A=${PWD#*github.com/};A=${A%/ffjson};cd ../..;mv $A pquerna;cd pquerna/ffjson
   - go get -d -v -t ./...
 
 script: make clean && make lint && make test && make test 
 
+jobs:
+  include:
+    - stage: Fuzz regression
+      go: 1.12.x
+      dist: bionic
+      script: ./fuzzit.sh local-regression
+
+    - stage: Fuzz
+      if: branch = master AND type IN (push)
```

Comment:
This does not do what you think it does. From the travis documentation:
```
require the branch name to be master (note for PRs this is the base branch name)
```
So this doesn't just run on pushes to master. It also runs to pushes to pull requests that want to be merged into master (so master is their base branch).

Reply:
I'm doing it like this: https://github.com/valyala/fasthttp/blob/master/.travis.yml#L56-L57